src/impl_/pyclass/lazy_type_object.rs

```diff
@@ -46,15 +46,30 @@ impl<T> LazyTypeObject<T> {
 
 impl<T: PyClass> LazyTypeObject<T> {
     /// Gets the type object contained by this `LazyTypeObject`, initializing it if needed.
+    #[cfg_attr(
```

Comment:
This is inside the `impl_` module so we don't need to worry about semver compliance and can assume all the callers are inside PyO3 and its macros.

So no need for the deprecation here, and we can probably even get away with just changing the type of `get_or_init` rather than adding `get_or_init_bound`.

Reply:
I thought that might be the case, but I wasn't certain. I'll fixup.